spec/javascript-spec.coffee

```diff
@@ -53,6 +53,28 @@ describe "Javascript grammar", ->
       {tokens} = grammar.tokenizeLine('with')
       expect(tokens[0]).toEqual value: 'with', scopes: ['source.js', 'keyword.control.js']
 
+    map =
```

Comment:
Can you make it so the test itself isn't looped, only the contents within the test?  So
```coffee
it "...", ->
  map =
    ...
  for keyword, scope of map
    ...
```

If necessary you can keep `map` out of the tests to avoid duplication.